src/openadr_ven/agent.py

```diff
@@ -259,6 +264,7 @@ def _parse_config(self, config_path: str) -> Dict:
         show_fingerprint = bool(config.get(SHOW_FINGERPRINT, True))
         ven_id = config.get(VEN_ID)
         disable_signature = bool(config.get(DISABLE_SIGNATURE))
+        demo_ven = config.get(DEMO_VEN)
```

Comment:
Add a default False to ensure it doesn't error out if 'DEMO_VEN' is missing.

Reply:
The `.get` method [returns `None`  by default if no key exists](https://docs.python.org/3/library/stdtypes.html#dict.get).

I also tested out installing this agent using an agent config with and without 'demo_ven' configuration and they both installed and worked as expected. I did not see an error when 'demo_ven' was missing, which is ok since it is an optional parameter.